boegelbot.py

```diff
@@ -512,7 +513,7 @@ def process_notifications(notifications, github, github_user, github_account, re
                 msg = mention_regex.sub(' ', comment_txt)
 
                 # require that @<host> is included in comment before taking any action
-                if host_regex.search(msg):
+                if host_regex.search(msg) or gpuhost_regex.search(msg):
```

Comment:
Won't this cause trouble if `--host` is specified to restrict access, but `--gpuhost` is not?

It seems like in that case, `gpuhost_regex` will just check for an `@` in the message, so it'll basically match any request, even if `--host` is used to restrict requests.

A solution for this could be including this before `gpuhost_regex` is defined above:

```python
if not gpuhost:
    gpuhost = 'NO_GPUHOST_PATTERN_PROVIDED
```

And to also cover the vice versa case (only `--gpuhost` specified, `--host` not), we should do the same for `host`?

Reply:
Good point :+1: 

I implemented that in https://github.com/boegel/boegelbot/pull/11/commits/cbb2b51363739e7efbe61351457e4acaa7511deb